modules/installation-configuration-parameters.adoc

```diff
@@ -689,6 +693,14 @@ Optional AWS configuration parameters are described in the following table:
 |Valid link:https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/EBSVolumeTypes.html[AWS EBS volume type],
 such as `io1`.
 
+|`controlPlane.platform.aws.rootVolume.kmsKeyARN`
```

Comment:
@xenolinux The top rows of this table are the params for compute nodes (`compute.*`), so I except `controlPlane.platform.aws.rootVolume.kmsKeyARN` presents after `controlPlane.platform.aws.iamRole` and before `controlPlane.platform.aws.type`

btw, the preview link does not work for me.
compute.platform.aws.rootVolume

Reply:
Addressed